.github/workflows/tests.yml

```diff
@@ -0,0 +1,37 @@
+name: Run tests
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [ 3.10.0 ]
+        django-version: [ "django32" ]
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Set Up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Display Python Version
```

Comment:
I believe this was only for debugging during the development. Since we can always verify python version from the logs so we can skip this step from the workflow.